client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -113,6 +124,24 @@ const EventForm: React.FC<EventFormProps> = (props) => {
   const watchSponsorsArray = watch('sponsors');
   const inviteOnly = watch('invite_only');
   const venueType = watch('venue_type');
+  const chapter = watch('chapter_id');
+
+  const {
+    loading: loadingVenues,
+    error: errorVenues,
+    data: dataVenues,
+  } = useChapterVenuesQuery({ variables: { chapterId: chapter } });
+
+  useEffect(() => {
+    resetField('chapter_id', { defaultValue: adminedChapters[0]?.id ?? -1 });
+  }, [adminedChapters]);
+
+  useEffect(() => {
+    resetField('venue_id', {
+      defaultValue:
+        defaultValues.venue_id ?? dataVenues?.chapterVenues[0]?.id ?? -1,
+    });
+  }, [dataVenues]);
```

Comment:
It's a bit out of scope of this PR, but this got me thinking: could we avoid resetting if we pulled the data fetching up into the parent compoenent? i.e. don't create the form until we have all the data and so avoid setting the wrong `defaultValue`.

Reply:
For venues problem would be fact venues belong to chapter. Unless all venues, for corresponding _possible_ admined chapters, are loaded before rendering form, there's bound to be setting default
when `dataVenues` changes, after selected chapter changed.

Reply:
That's a shame, but it sounds like some amount of resetting is inevitable.  Oh well.